revscoring/utilities/tune.py

```diff
@@ -124,6 +127,12 @@ def main(argv=None):
     if args['--scale']:
         additional_params['scale'] = args['--scale'],
 
+    additional_params['multilabel'] = False
+    if args['--multilabel']:
+        _, all_labels = zip(*value_labels)
+        labels = get_multilabel_set(all_labels)
```

Comment:
Use the `--labels`, weights or rates just like in multiclass.

Reply:
https://github.com/wiki-ai/revscoring/pull/376/files#diff-34e19f77339293b2504629af65db696cR117